omod/src/main/java/org/openmrs/web/servlet/LoginServlet.java

```diff
@@ -65,18 +68,19 @@
 	 * @see javax.servlet.http.HttpServlet#doPost(javax.servlet.http.HttpServletRequest,
 	 *      javax.servlet.http.HttpServletResponse)
 	 */
+
 	@Override
 	protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
 		HttpSession httpSession = request.getSession();
-		
+		Integer loginAttemptsByUser;
 		String ipAddress = request.getRemoteAddr();
 		Integer loginAttempts = loginAttemptsByIP.get(ipAddress);
 		if (loginAttempts == null) {
 			loginAttempts = 1;
 		}
 		
 		loginAttempts++;
-		
+		loginAttemptsByUser = loginAttempts - 1;
```

Comment:
What's the difference btn loginAttemptsByUser and loginAttempts? Are there attempts that aren't made by a User?

Reply:
i intended to differentiate between  login by User name , and login by IP address

Reply:
Can you improve the naming to reflect that?

Reply:
Alryt sam

Reply:
i updated it